ecs-cli/modules/commands/flags/flags.go

```diff
@@ -118,6 +118,8 @@ const (
 	UpdateExistingSecretsFlag = "update-existing-secrets"
 	RoleNameFlag              = "role-name"
 	NoRoleFlag                = "no-role"
+	NoOutputFileFlag          = "no-output-file"
```

Comment:
Can you remind me if this was the final name of the flag? For some reason I thought there was some discussion about how "output file" was a bit vague.